netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -301,6 +301,8 @@ protected void onStreamError(ChannelHandlerContext ctx, Throwable cause,
     NettyClientStream stream = clientStream(connection().stream(http2Ex.streamId()));
     if (stream != null) {
       stream.transportReportStatus(Utils.statusFromThrowable(cause), false, new Metadata());
+    } else {
+      logger.log(Level.INFO, "Stream error for unknown stream " + http2Ex.streamId(), cause);
```

Comment:
As discussed offline, it seems this might log in "normal" conditions when clients cancel, and it would take some effort to figure out how that would impact users. I have no concern if at `FINE` level.